torchci/pages/flakytest.tsx

```diff
@@ -27,13 +27,13 @@ export default function Page() {
     <div>
       <h1>PyTorch CI Flaky Tests</h1>
       <h2>
-        Test Name: <code>{name}</code>
+        Test Name Filter: <code>{name === "%" ? "<any>" : name}</code>
```

Comment:
not sure if I like this <any> but I guess it's fine. Also the code thing makes me think it's not a filter. I would prefer a text input here, even though we kinda use this pattern on the main page.